# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextEAGL.h
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 gfx/gl/GLContextEAGL.h
--- a/gfx/gl/GLContextEAGL.h	Fri Jun 12 21:53:14 2020 +0000
+++ b/gfx/gl/GLContextEAGL.h	Mon Jun 15 18:25:55 2020 +0000
@@ -22,9 +22,8 @@
 
  public:
   MOZ_DECLARE_REFCOUNTED_VIRTUAL_TYPENAME(GLContextEAGL, override)
-  GLContextEAGL(CreateContextFlags flags, const SurfaceCaps& caps,
-                EAGLContext* context, GLContext* sharedContext,
-                bool isOffscreen, ContextProfile profile);
+  GLContextEAGL(const GLContextDesc&, EAGLContext* context,
+                GLContext* sharedContext, ContextProfile profile);
 
   ~GLContextEAGL();
 